src/cat-data-service/src/service/common/objects/delegator_address.rs

```diff
@@ -0,0 +1,22 @@
+use poem_openapi::{types::Example, Object};
+
+#[derive(Object)]
+pub(crate) struct DelegatorAddress {
+    #[oai(validator(pattern = "[0-9a-f]{64}"))]
```

Comment:
All hex strings should start with 0x
```suggestion
    #[oai(validator(pattern = "0x[0-9a-f]{64}"))]
```